src/architect/account/account.utils.ts

```diff
@@ -70,6 +84,7 @@ export default class AccountUtils {
       }
     } else {
       inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
+      const can_run_local: boolean = await this.shouldListLocalAccounts();
```

Comment:
You don't need to specify the type here because the function return specifies. 
```ts
const can_run_local = await this.shouldListLocalAccounts();
```

Reply:
removed specified type from can_run_local